### PR TITLE
Chore/use legacy cleanup [OR-68404]

### PR DESCRIPTION
**Pull request by alexdudescu**
- implement `SettingsStore` to persist playground configured settings
- drop `useLegacy` flag

[OR-68404](https://uipath.atlassian.net/browse/OR-68404)

[OR-68404]: https://uipath.atlassian.net/browse/OR-68404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: projects/angular/components/ui-grid/src/components/ui-grid-toggle-columns/ui-grid-toggle-columns.component.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
     </button>
 
     <mat-select [value]="selected"
                 [aria-label]="togglePlaceholderTitle ?? ''"
                 [disableOptionCentering]="true"
-                [class.use-alternate]="!useLegacyDesign"
-                [panelClass]="['ui-grid-toggle-panel', useLegacyDesign ?  '' : 'ui-grid-toggle-panel-alternate']"
+                [panelClass]="['ui-grid-toggle-panel', 'ui-grid-toggle-panel-alternate']"
                 (selectionChange)="selectionChange($event)"
+                class="use-alternate"
```
Comment (anbalase) [inline projects/angular/components/ui-grid/src/components/ui-grid-toggle-columns/ui-grid-toggle-columns.component.html:17]: could we apply the styling by default and drop the class?

File: projects/angular/components/ui-grid/src/ui-grid.component.html [hunk 2]
```diff
@@ -549,19 +541,10 @@
...
-        <ng-container *ngTemplateOutlet="multiPageSelectionInfo"></ng-container>
-    </div>
-</ng-template>
-
 <ng-template #multiPageSelectionRow>
     <div *ngIf="!isProjected &&
                 showMultiPageSelectionInfo"
-         class="ui-grid-row ui-grid-selection-info-container"
-         [class.ui-grid-selection-info-container-alternate]="!useLegacyDesign">
+         class="ui-grid-row ui-grid-selection-info-container ui-grid-selection-info-container-alternate">
```
Comment (anbalase) [inline projects/angular/components/ui-grid/src/ui-grid.component.html:547]: same comment, alternate no longer makes sense

File: projects/angular/components/ui-grid/src/ui-grid.component.html [hunk 3]
```diff
@@ -720,8 +671,7 @@
                             [resetToDefaults]="intl.toggleResetToDefaults"
                             [toggleTitle]="intl.toggleTitle"
                             [togglePlaceholderTitle]="intl.togglePlaceholderTitle"
-                            [useLegacyDesign]="useLegacyDesign"
-                            [showDivider]="!useLegacyDesign && (displayToggleColumnsDivider$ | async) ?? false"
+                            [showDivider]="!false && (displayToggleColumnsDivider$ | async) ?? false"
```
Comment (anbalase) [inline projects/angular/components/ui-grid/src/ui-grid.component.html:674]:
```suggestion
                            [showDivider]="(displayToggleColumnsDivider$ | async) ?? false"
```

File: projects/angular/components/ui-grid/src/ui-grid.component.html [hunk 4]
```diff
@@ -648,7 +599,7 @@
 <ng-template #inlineFiltersTmpl>
     <ng-container *ngFor="let column of columns$ | async">
         <div *ngIf="column.dropdown?.visible$ | async"
-             [ngClass]="{'ui-grid-alternate-filter-container': collapsibleFilters}"
+             [attr.data-collapsed]="!collapseFiltersCount || null"
```
Comment (alexdudescu) [inline projects/angular/components/ui-grid/src/ui-grid.component.html:602]: This class was previously only added for test element query-ing, so it was factored to a data attribute

**Review by dan-mirescu — APPROVED** [review]

**Pull request merged**